src/index.njk

```diff
@@ -68,6 +64,20 @@ masthead: true
         <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
       </div>
 
+      <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
+        <h2 class="govuk-heading-l">How we work</h2>
+        <p class="govuk-body">
+        The GOV.UK Design System helps teams follow the <a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a> and <a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a>. This website also follows the <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">Government Digital Service (GDS) style guide</a> and <a class="govuk-link" href="https://www.gov.uk/guidance/content-design">content design guidance.</a>
+        </p>
+
+        <p class="govuk-body">
+<a class="govuk-link" href="/community/accessibility-strategy/">Our accessiblity strategy</a> outlines our principles and work to improve the accessibility within the Design System.</p>
```

Comment:
Should we say 'within the GOV.UK Design System' instead of 'within the Design System', for consistency?

Reply:
```suggestion
<a class="govuk-link" href="/community/accessibility-strategy/">Our accessiblity strategy</a> outlines our principles and work to improve the accessibility within the GOV.UK Design System.</p>
```